### DISCUSSION TITLE
status 1 on windows 11

### DISCUSSION THREAD

**Discussion by thomasbnt** [coreybutler/nvm-windows#834] [Support] [answered]
Hello ! :wave: 

I've got the **status 1** error, I'm on Windows 11. I [followed this guide on the wiki](https://github.com/coreybutler/nvm-windows/wiki/Common-Issues#permissions-exit-1-exit-5-access-denied-exit-145), but have same issue again.

```bash
PS > nvm install 16.0.0
Downloading node.js version 16.0.0 (64-bit)... 
Complete
Creating ...\AppData\Roaming\nvm\temp

Downloading npm version 7.10.0... Complete
Installing npm v7.10.0...

Installation complete. If you want to use this version, type

nvm use 16.0.0
PS > nvm use 16.0.0
exit status 1: Acc�s refus�.
```

**Comment by awdr74100** [comment, marked as answer]
Use sudo directly and save yourself a lot of trouble:

```shell
winget install -e --id gerardog.gsudo
```

When you want to switch versions:

```shell
sudo nvm use 16.0.0
```
